spec/support/features/idv_helper.rb

```diff
@@ -94,11 +94,9 @@ def click_idv_address_choose_usps
   end
 
   def choose_idv_otp_delivery_method_sms
-    page.find(
-      'label',
-      text: t('devise.two_factor_authentication.otp_delivery_preference.sms')
-    ).click
-    click_on t('idv.buttons.send_confirmation_code')
+    using_wait_time(5) do
+      click_on t('idv.buttons.send_confirmation_code')
+    end
```

Comment:
SMS is the default option. No need to explicitly click on it. This change also allows us to test that it is indeed the default option.